CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 3.4.0
+
 * Add support for flash messages
```

Comment:
I forgot to add the changelog message for https://github.com/alphagov/govuk_admin_template/pull/101

Could you add:

```
* Add optional signout element to navigation bar 
```